src/main/java/core/basesyntax/service/impl/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.TransactionHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private OperationStrategy strategy = new OperationStrategy();
```

Comment:
Please use dependency injection via constructor instead of field initialization. It will make code testing and maintaining more simple